Studio-Lovelies/GG-JointJustice-Unity
# Open DevKit location across all platforms; always reload on file changes, even after errors

## Summary

Contains 3 changes. A summary and testing instructions for each:

- The DevKit should always open an OS-specific file browser window to show the location of the DevKit assets. This incorrectly only worked on Windows and is resolved by 488608f3d989a865f7717f083018a1afb2cece65.
  ![226142481-5dd7254a-43ab-4f0d-8823-f56619dfe055](https://github.com/user-attachments/assets/6c46a2a2-9216-4335-86d0-42c7e2b50b19)

- 47b85e99bf90aed0abbff0644d6dda44731aeb37 should close #478.

- 3edbbe859eb36db431b3a9101cb80e98be7418f0 makes the new Prerelease workflow work. Hopefully.

## Testing instructions
- Follow [the wiki guide to the DevKit](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/wiki/Scripting-Basics-using-Inky) and notice a file browser opening after pressing the shortcut

- See #478.

- You know the drill. 🙏 

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - closes #478
